saleor/graphql/discount/types.py

```diff
@@ -36,10 +37,12 @@ class Sale(CountableDjangoObjectType):
         SaleTranslation,
         language_code=graphene.Argument(
             LanguageCodeEnum,
-            description="A language code to return the translation for.",
+            description=TranslationFieldDecriptions.LANGUAGE_CODE.format(
```

Comment:
Let's try to refactor it event more to have a custom `TranslationField` that extends the `graphene.Field` and takes three arguments: type, type name (to use in field descriptions) and a resolver.

Would be great if we could replace this piece:

```
translation = graphene.Field(
        SaleTranslation,
        language_code=graphene.Argument(
            LanguageCodeEnum,
            description=TranslationFieldDecriptions.LANGUAGE_CODE.format(
                type_name="sale"
            ),
            required=True,
        ),
        description=TranslationFieldDecriptions.DESCRIPTION.format(type_name="sale"),
        resolver=resolve_translation,
    )
```

with something like:

```
translation = TranslationField(SaleTranslation, type_name="sale", resolver=resolve_translation)
```

Maybe even `resolver` arg could be optional and by default it could expect that there is a `resolve_translation` function in the type.